src/app/client/commitsContainer.html

```diff
@@ -3,8 +3,11 @@
   display: none;
 }
 </style>
-
-<link type="text/css" rel="stylesheet" href="{{resourcePath}}css/commitStream.css">
+<!-- Since this file is for cs classic, we will still point to light-->
+<!-- doesn't this force us to get styles from  cs instance still for classic?
+This brings up another problem.  The
+-->
+<link type="text/css" rel="stylesheet" href="{{resourcePath}}css/commitStream-Light.css">
```

Comment:
What happens with this stylesheet?

Reply:
resource path is on the Commitstream server.  It then gets loaded and send down to core in the case of the the OG commistream users as apposed to our newer way of getting styles from core.